Core/Src/torque_manager.c

```diff
@@ -7,109 +7,111 @@
 
 #include "torque_manager.h"
 #include "cmsis_os.h"
-#include "global_definitions.h"
+#include "global_variables.h"
 #include "DynamicControls/lateral_control.h"
 
-extern volatile uint16_t throttle_percent;
+extern volatile float g_wheel_speed[4];
 extern volatile uint16_t steering_wheel;
+extern volatile uint8_t internal_wheel;
+extern volatile uint16_t gyro_yaw; // ainda nao existe
+extern volatile uint16_t throttle_percent;
 extern modos modo_selecionado;
 extern osMessageQueueId_t q_ref_torque_messageHandle;
 extern osMutexId_t m_state_parameter_mutexHandle;
 
 void torque_manager(void *argument) {
 
-	ref_torque_t ref_torque_message;
+    ref_torque_t ref_torque_message;
+
+    for (;;) {
+        uint32_t tick;
+        tick = osKernelGetTickCount();
```

Comment:
pode simplificar pra:
```suggestion
        uint32_t tick = osKernelGetTickCount();
```
(dá na mesma, mas é mais usual dessa forma)